src/devtools/client/debugger/src/components/SourceOutline/SourceOutline.tsx

```diff
@@ -187,12 +188,14 @@ export default function SourceOutlineWrapper() {
   );
 
   return (
-    <Suspense fallback={null}>
-      <SourceOutline
-        cursorPosition={cursorPosition || null}
-        selectedSource={selectedSource || null}
-        symbols={symbols}
-      />
-    </Suspense>
+    <ErrorBoundary>
```

Comment:
Just curious, but is this new boundary a redundant fix (given the change to check error code 48 in the cache)?

Probably still a good idea FWIW 👍🏼  Just curious

Reply:
It's probably redundant, and tbh I'm still confused why the boundary we have around the root of the app isn't catching this 🤷‍♂️ .

But this is also probably one of those cases where it doesn't hurt to have the boundary there anyway.